app/Models/ProviderList.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * @property int $id
+ * @property string $name
+ * @property string $url
+ * @property string $color
+ */
+class ProviderList extends Model
+{
+    public function provider()
+    {
+        return $this->belongsTo(Provider::class);
+    }
+}
```

Comment:
Do we need that model?

Reply:
It's mapped ProviderList table with all (14 - for now) providers.

Reply:
Providers table "connects" cities and providers:
- id,
- provider_id,
- city_id,
- timestamps.